apps/analysis/views.py

```diff
@@ -45,6 +48,38 @@ def analysis_details(request, team_slug: str, pk: int):
     )
 
 
+@login_and_team_required
+@permission_required("analysis.change_analysis", raise_exception=True)
+def analysis_configure(request, team_slug: str, pk: int):
```

Comment:
No need to change this, but using a view class would make this a bit cleaner imo

Reply:
Can you say more? How would a view class make it clearer?